thentos-core/src/Thentos/Prelude.hs

```diff
@@ -0,0 +1,33 @@
+module Thentos.Prelude (module X) where
+
+import Control.Applicative as X
+import Control.Conditional as X hiding ((|>), (<|), (??))
+-- import Control.Exception.Lifted as X hiding (try)
```

Comment:
Shouldn't be here.
I tried to make it part of Prelude but this would require to be a real Prelude, namely override the original one. Advantage: simpler to get rid of old version of functions such as this `try`, disadvantage: slightly longer to import this prelude as we have to either `import Prelude ()` or use `NoImplicitPrelude` option.